quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/KeycloakProcessor.java

```diff
@@ -272,7 +273,18 @@ private void configureDefaultPersistenceUnitProperties(ParsedPersistenceXmlDescr
         Properties unitProperties = descriptor.getProperties();
 
         unitProperties.setProperty(AvailableSettings.DIALECT, config.defaultPersistenceUnit.dialect.dialect.orElse(null));
-        unitProperties.setProperty(AvailableSettings.JPA_TRANSACTION_TYPE, PersistenceUnitTransactionType.JTA.name());
+        String storage = Configuration.getRawValue(
```

Comment:
We can use this option to choose between `JTA` and `RESOURCE_LOCAL`. See https://github.com/keycloak/keycloak/blob/a76b597b35a4d9ce9015016db6f02793e4d03ea5/quarkus/config-api/src/main/java/org/keycloak/config/TransactionOptions.java#L12.

Reply:
Included in the latest commit